task/sbom-json-check/0.2/README.md

```diff
@@ -0,0 +1,37 @@
+# sbom-json-check task
+
+## Deprecation notice
+
+This task is deprecated, please remove it from you pipeline.
```

Comment:
Can you add a sentence where for when the task will start to cause EC failures (well, it should only cause EC failures for non-TA pipelines... )?

Reply:
THat should start happening once the EC policy hit the deprecation date, let me first synch with EC about that date.